package.json

```diff
@@ -14,10 +14,10 @@
     "lint": "eslint ./packages --ext .js"
   },
   "devDependencies": {
-    "eslint-config-deloitte": "^3.3.1",
-    "lerna": "^2.11.0",
-    "mocha": "^5.1.1",
-    "rimraf": "^2.6.2",
-    "stylelint-config-deloitte": "^1.0.2"
+    "eslint-config-deloitte": "3.3.1",
+    "lerna": "2.11.0",
+    "mocha": "5.1.1",
+    "rimraf": "2.6.2",
+    "stylelint-config-deloitte": "1.0.2"
```

Comment:
Pinning these development dependencies in the top level makes sense, because it makes it clear what versions of dependencies we are using with a quick glance at the package.json file.